app/src/main/java/com/wafflestudio/snutt2/views/logged_in/home/search/search_option/TagsColumn.kt

```diff
@@ -0,0 +1,103 @@
+package com.wafflestudio.snutt2.views.logged_in.home.search.search_option
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.State
+import androidx.compose.runtime.derivedStateOf
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.unit.dp
+import com.wafflestudio.snutt2.components.compose.VividCheckedIcon
+import com.wafflestudio.snutt2.components.compose.VividUncheckedIcon
+import com.wafflestudio.snutt2.components.compose.clicks
+import com.wafflestudio.snutt2.lib.Selectable
+import com.wafflestudio.snutt2.model.TagDto
+import com.wafflestudio.snutt2.ui.SNUTTColors
+import com.wafflestudio.snutt2.ui.SNUTTTypography
+
+@Composable
+fun TagsColumn(
+    modifier: Modifier,
+    tagsByTagType: List<Selectable<TagDto>>,
+    selectedTimes: State<List<List<Boolean>>>,
+    baseAnimatedFloat: State<Float>,
+    onToggleTag: (TagDto) -> Unit,
+    openTimeSelectSheet: () -> Unit,
+) {
+    val configuration = LocalConfiguration.current
+    val context = LocalContext.current
+    val alphaAnimatedFloat by remember {
+        derivedStateOf { 1f - baseAnimatedFloat.value }
+    }
+    val offsetXAnimatedDp by remember {
+        derivedStateOf {
+            val tagsColumnWidth =
+                configuration.screenWidthDp - SearchOptionSheetConstants.TagColumnWidthDp
+            return@derivedStateOf tagsColumnWidth.dp * baseAnimatedFloat.value
+        }
+    }
```

Comment:
여기도 derivedState 빼도 되지?
```suggestion
    val alphaAnimatedFloat = 1f - baseAnimatedFloat.value
    val offsetXAnimatedDp = (configuration.screenWidthDp - SearchOptionSheetConstants.TagColumnWidthDp).dp * baseAnimatedFloat.value
```

Reply:
여기도있었네

Reply:
이런 린트가..